pkg/harvester/edit/harvesterhci.io.virtualmachineimage.vue

```diff
@@ -184,15 +209,6 @@ export default {
   },
 
   watch: {
-    'value.metadata.namespace'(neu) {
```

Comment:
This is a bug that changing created image ns should not update `securityParameters.sourceImageNamespace`.